test/livebook_teams/web/apps_live_test.exs

```diff
@@ -384,134 +384,5 @@ defmodule LivebookWeb.Integration.AppsLiveTest do
       assert render(view) =~ "Ungrouped apps"
       assert render(view) =~ slug
     end
-
-    test "filter the apps based on slug, name and app folder",
```

Comment:
Since there's no LiveView test for that, what about adding a JS test? So we can ensure it will filter the apps based on the search criteria

Reply:
@aleDsz yes, that's [what I'm thinking about doing](https://github.com/livebook-dev/livebook/pull/3094#issuecomment-3534199375), maybe with https://github.com/ftes/phoenix_test_playwright or https://github.com/elixir-wallaby/wallaby

Reply:
The hooks directory actually only has unit tests for some specific encapsulated classes that we use from hooks. I don't think there's an easy way to unit test it, because it requires the page structure.

But I don't think adding browser testing for this specifically is worth it either.